src/controllers/commands/CommandController.cpp

```diff
@@ -663,9 +663,8 @@ void CommandController::initialize(Settings &, Paths &paths)
                 (!channel->isTwitchChannel() || channel->isEmpty()))
             {
                 channel->addMessage(makeSystemMessage(
-                    "Usage: /streamlink <channel>. You can also use the "
-                    "command without arguments in any Twitch channel to open "
-                    "it in streamlink."));
+                    "Usage: /streamlink [channel]. If no channel argument is "
+                    "specified, it will open the current Twitch channel."));
```

Comment:
I guess we can preserve the wording to keep it obvious what will happen, this is a nitpick but I feel like just "will open current channel" is missing something. 
```suggestion
                    "Usage: /streamlink [channel]. If no channel argument is "
                    "specified, it will open the current Twitch channel in streamlink."));
```

Reply:
**BASED** on this suggestion, and more chat offline, we decided on `/streamlink [channel]. Open specified Twitch channel in streamlink. If no channel argument is specified, open the current Twitch channel instead.`